man/systemd-sbsign.xml

```diff
@@ -85,11 +99,44 @@
         <xi:include href="version-info.xml" xpointer="v257"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--signed-attributes=<replaceable>PATH</replaceable></option></term>
+        <term><option>--signed-attributes-signature=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>Configure the signed attributes blob (as provided by
+        <command>export-signed-attributes</command>) and corresponding signature for the
+        <command>sign</command> command.</para>
+
+        <xi:include href="version-info.xml" xpointer="v258"/></listitem>
+      </varlistentry>
+
       <xi:include href="standard-options.xml" xpointer="help"/>
       <xi:include href="standard-options.xml" xpointer="version"/>
     </variablelist>
   </refsect1>
 
+
+  <refsect1>
+    <title>Examples</title>
+
+    <example>
+      <title>Offline EFI secure boot signing of a PE binary</title>
+
+      <para>The following does offline secure boot signing of systemd-boot:</para>
+
+      <programlisting>SD_BOOT="$(find /usr/lib/systemd/boot/efi/ -name "systemd-boot*.efi" | head -n1)"
+/usr/lib/systemd/systemd-sbsign export-signed-attributes --certificate cert.crt --output sattrs.bin "$SD_BOOT"
```

Comment:
`.bin`? maybe name this `.p7<something>` instead? this is after all in pkcs7 format if i get this right, except that it lacks the sig itself?